net/rhp/contracts.go

```diff
@@ -117,29 +112,29 @@ func ValidateContractRenewal(existing, renewal types.FileContract, currentHeight
 	case renewal.RevisionNumber != 0:
 		return errors.New("revision number must be zero")
 	case renewal.Filesize != existing.Filesize:
-		return errors.New("file size must not change")
+		return errors.New("filesize must not change")
 	case renewal.FileMerkleRoot != existing.FileMerkleRoot:
-		return errors.New("file merkle root must not change")
+		return errors.New("file Merkle root must not change")
 	case renewal.WindowEnd < existing.WindowEnd:
-		return errors.New("renewal window end must be after existing window end")
+		return errors.New("renewal window end must be after current window end")
 	case renewal.WindowStart < currentHeight+settings.WindowSize:
 		return errors.New("contract ends too soon to safely submit the contract transaction")
 	case renewal.WindowStart > currentHeight+settings.MaxDuration:
 		return errors.New("contract duration is too long")
 	case renewal.WindowEnd < renewal.WindowStart+settings.WindowSize:
 		return errors.New("proof window is too small")
-	case renewal.ValidHostOutput.Address != settings.Address:
-		return errors.New("wrong address for host valid output")
-	case renewal.MissedHostOutput.Address != settings.Address:
-		return errors.New("wrong address for host missed output")
-	case renewal.ValidHostOutput.Value.Cmp(settings.ContractFee) < 0:
+	case renewal.HostOutput.Address != settings.Address:
+		return errors.New("wrong address for host output")
+	case renewal.HostOutput.Value.Cmp(settings.ContractFee) < 0:
```

Comment:
Should this check `TotalCollateral` too? 🤔 
```go
renewal.HostOutput.Value.Cmp(settings.ContractFee.Add(renewal.TotalCollateral)) < 0
```

Reply:
yeah I wasn't sure about this one, because renewal math. I *think* what you wrote it correct, though. Or should it be exactly equal, as in `ValidateContractFormation`?

Reply:
It won't be exactly equal because we don't know the additional storage cost